modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -0,0 +1,389 @@
+////
+This module included in the following assemblies:
+-distr_tracing_otel/distr-tracing-otel-configuring.adoc
+////
+:_content-type: REFERENCE
+[id="distr-tracing-config-otel-collector_{context}"]
+= OpenTelemetry Collector configuration options
+
+:FeatureName: The {OTELName} Operator
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+The OpenTelemetry Collector consists of three components that access telemetry data:
+
+* *Receivers* - A receiver, which can be push or pull based, is how data gets into the Collector. Generally, a receiver accepts data in a specified format, translates it into the internal format, and passes it to processors and exporters defined in the applicable pipelines. By default, no receivers are configured. One or more receivers must be configured. Receivers may support one or more data sources.
+
+* *Processors* - (Optional) Processors run through the data between it is received and exported. By default, no processors are enabled. Processors must be enabled for every data source. Not all processors support all data sources. Depending on the data source, multiple processors might be enabled. In addition, it is important to note that the order of processors matters.
+
+* *Exporters* - An exporter, which can be push or pull based, is how you send data to one or more back ends or destinations. By default, no exporters are configured. One or more exporters must be configured. Exporters can support one or more data sources. Exporters might be used with their default settings, but many exporters require configuration to specify at least the destination and security settings.
+
+You can define multiple instances of components in a custom resource YAML file. When configured, these components must be enabled through pipelines defined in the `spec.config.service` section of the YAML file. As a best practice, only enable the components that you need.
+
+.OpenTelemetry Collector custom resource file
+====
```

Comment:
This renders in a box because of `====`
Consider removing it

Reply:
Thank you. Already removed in response to an earlier, global comment.